openapi/file360/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Content & Files/Documents", "Cost/Freemium"]
+org = "ballerinax"
+name = "file360"
```

Comment:
please add the following line
icon = "icon.png"

Reply:
Added as sugested.